controller/userController.js

```diff
@@ -8,43 +10,46 @@ exports.login = async (req, res) => {
     try {
         const user = await User.findOne({ username: username });
         if (!user) {
-            res.status(401).json({ err: "A user with this email could not be found." });
+            res.status(401).json({ err: "A user with this username could not be found." });
         }
-        if (user.password !== password)
+        const comparePass = await bcrypt.compare(password, user.password);
+        if (!comparePass)
```

Comment:
a minor issue - please use {} event if you have 1 line in if statement as follows:
```suggestion
        if (!comparePass) {
            res.status(401).json('Wrong password')
        }
```